operators/trim_left_or_right_handles.py

```diff
@@ -65,3 +75,19 @@ def select_strip_handle(self, sequences, side, frame):
                 s.select = False
             if handle_side:
                 bpy.ops.sequencer.select_handles(side=handle_side)
+
+    def ripple_sequences(self, sequence, context):
+        gap = 0
+        side = 'RIGHT' if sequence.select_right_handle else 'LEFT'
+        if side == 'RIGHT':
+            gap = sequence.frame_final_start + sequence.frame_duration - sequence.frame_final_end
+        else:
+            gap = abs(sequence.frame_start - sequence.frame_final_start)
+
+        for s in context.sequences:
+            if s == sequence or s.channel != sequence.channel:
+                continue
+            if side == 'RIGHT' and sequence.frame_final_end < s.frame_final_start:
+                s.frame_start -= gap
+            elif side == 'LEFT' and sequence.frame_final_start > s.frame_final_end:
```

Comment:
No that's not rippling. You're moving strips before the trim. Rippling is always to the right side, as I showed you in the recording here. You want to ripple = remove the exact gap you create when trimming on the trimmed strips and all the ones after it. Check the video again, I showed you an example for both the right and left trim iirc.